src/python/pants/backend/python/goals/package_pex_binary.py

```diff
@@ -53,8 +56,26 @@ class PexBinaryFieldSet(PackageFieldSet, RunFieldSet):
     zip_safe: PexZipSafeField
     platforms: PythonPlatformsField
     unzip: PexUnzipField
+    execution_mode: PexExecutionModeField
```

Comment:
You can name this field whatever you want - it's only the type hint that matters for `FieldSet`. Probably better to name this with `_`, and make the property be public.

Reply:
Why does this one get _ and no other? I'm lost.

Reply:
Aha - gotcha. Well the property isn't intended to be public. Its a detail of the implementation of the (rightly) public `generate_additional_args`.